product_nomenclature/__manifest__.py

```diff
@@ -4,10 +4,10 @@
 
 {
     "name": "Product nomenclature",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "author": "Creu Blanca",
     "category": "Product",
-    "depends": ["product", "sales_team"],
+    "depends": ["product", "sales_team", "sale"],
```

Comment:
If you put sale, you can remove, product and sales_team

Reply:
Done!